airbyte-commons/src/main/java/io/airbyte/commons/version/AirbyteVersion.java

```diff
@@ -133,7 +134,7 @@ public boolean checkOnlyPatchVersionIsUpdatedComparedTo(final AirbyteVersion ano
   }
 
   public boolean isDev() {
-    return version.equals(DEV_VERSION);
+    return version.equals(DEV_VERSION) || version.contains(OSS_BRANCH_VERSION_PREFIX);
```

Comment:
instead of `oss-branch-<whatever>` would it makes sense to just do `dev-<whatever`? i haven't thought through the tradeoffs here. you're adding a new magic word. if it has a significantly different meaning than dev it is worth having the name be separate. if they are the same conceptually then maybe repurposing dev to be either the full branch version or just a prefix seems more intuitive?

Reply:
I think that's a good idea - we'd still need to change the `version.equals(DEV_VERSION)` to `version.contains(DEV_PREFIX)` but that feels better to me than a whole other magic word, as you pointed out.

Reply:
agreed.